specs/www.rst

```diff
@@ -413,6 +414,9 @@ with the close code passed in the message (or 1000 if none is specified).
 * ``code`` (*int*) -- The WebSocket close code, as per the WebSocket
   spec.  Optional; if missing defaults to ``1000``.
 
+* ``reason`` (*Unicode string*) -- A reason given for the closure, can
+  be any string. Optional; if missing default is empty string.
```

Comment:
Given the type declared, should this be "if missing or None"?

Reply:
I've gone for the same wording as the [`query_string`](https://github.com/django/asgiref/blame/master/specs/www.rst#L94), should I change both?

Reply:
Yes, let's update both, explicit is better here I think.

Reply:
Done.